drakrun/drakrun/draksetup.py

```diff
@@ -83,6 +83,54 @@ def check_root():
         return True
 
 
+@click.command(help='Cleanup the changes made by draksetup')
+def cleanup():
+    if not check_root():
+        return
+
+    install_info = InstallInfo.try_load()
+
+    if install_info is None:
+        logging.error("The cleanup has been performed")
+        return
+
+    backend = get_storage_backend(install_info)
+    vm_ids = get_all_vm_conf()
+
+    for vm_id in vm_ids:
+        delete_vm_conf(vm_id)
```

Comment:
I think we may simply glob over config dir and delete everything

Reply:
I was keeping track of everything(network, storage, etc.) and deleting it for every VM, that's why for same modularity in the codebase, I was deleting the config files also one by one. This can help in extending the cleanup function(if needed) with parameters which allow deleting specific VM data only in the near future